platform/x11/cog-platform-x11.c

```diff
@@ -899,8 +899,10 @@ cog_x11_platform_setup(CogPlatform *platform, CogShell *shell G_GNUC_UNUSED, con
         return FALSE;
     }
 
-    if (!init_keyboard(error))
-        return FALSE;
+    if (!init_keyboard(error)) {
```

Comment:
**minor**: This could be squashed into https://github.com/Igalia/cog/pull/679/commits/27c30fbbd711a5fb13174b8c03af657a90701932  since affects to code added in this previous commit from this same PR.

Reply:
Done, squashed.